### PR TITLE
Installation docs experiments DOC-7864

### PR DESCRIPTION
**Pull request by paperclypse**
A bold experiment!

I have:
1. Updated the homepage to add an "Install New Relic" doc to the left nav and changed the quickstarts button to "Install New Relic".
2. I radically restructured the Install New Relic Doc.
3. I straight-up deleted the Get started with Full Stack Observability doc and redirected it to "Install New Relic".
4. I tweaked the guided install doc so it's less about a call to action and more a reference/explanation of what guided install is.

### CODE REVIEW

File: src/content/docs/new-relic-one/install-configure/install-new-relic.mdx [hunk 1]
```diff
@@ -22,37 +22,154 @@ redirects:
...
 <img
-  title="Quickstarts landing page"
-  alt="A screenshot of the quickstarts"
+  title="Instant observability landing page"
+  alt="A screenshot of the Instant Observability landing page"
   src={installNrQuick}
 />
 
-For specific install procedures of our monitoring services, check these links:
+Our [Instant Observability (I/O)](https://newrelic.com/instant-observability/) catalog is a showcase of everything you can monitor with New Relic. Not only that, click **+Install quickstart** anywhere to be funneled into the right installation flow, once you've [signed up for a free New Relic account](https://newrelic.com/signup).
```
Comment (akristen) [inline src/content/docs/new-relic-one/install-configure/install-new-relic.mdx:160]: showcases

Comment (akristen) [inline src/content/docs/new-relic-one/install-configure/install-new-relic.mdx:160]: showcases

**Review by akristen — DISMISSED** [review]
Left some comments as Codemarks. All of the changes are great—love the style + tone, which is conversational and polished. I left a tentative comment about the "balance" of the page, re: white space-to text-to images. It looks a little heavy in the middle, and I wonder if we can remove the I/O screenshot. Otherwise it's a massive improvement + I'm going to send my approval. :)

**Comment by paperclypse** [comment]
Got my approval a few commits ago :D

**Pull request merged**